### PR TITLE
Navbar tests added

### PR DESCRIPTION
**Pull request by Scott-Guinn**
Disabled the unwritten tests, adjusted some of the test configs to make them more dev friendly.

No longer displaying test coverage on every run but the code can be commented back in to view the coverage.

Wrote four very basic tests for NvisionNavbar component.

### CODE REVIEW

**Review by taylorsmart — APPROVED** [review]
Looks good - no comments or adjustments.

File: _tests_/back-end/success.test.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
-import {createMocks} from 'node-mocks-http';
-import success from '../pages/api/success';
+// import {createMocks} from 'node-mocks-http';
+// import success from '../pages/api/success';
 
-describe('/api/success', () => {
```
Comment (emmaknor) [inline _tests_/back-end/success.test.js:4]: thank you for this fix!

File: _tests_/front-end/components/login.test.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import '@testing-library/jest-dom';
 import * as React from 'react';
 import Login from 'login.jsx';
 
-describe('', () => {
+xdescribe('', () => {
```
Comment (emmaknor) [inline _tests_/front-end/components/login.test.js:6]: thank you again for all the clean up! i appreciate it!

File: _tests_/front-end/components/nvisionNavbar.test.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import '@testing-library/jest-dom';
+import * as React from 'react';
+import {GlobalStateProvider} from 'globalState.js';
+import NvisionNavbar from 'nvisionNavbar.jsx';
+
+describe('Nvision Nabar testing suite', () => {
+  const renderComponent = () => {
+    // this wrapper argument wraps the NvisionNavbar component in the
+    // GlobalStateProvider context.
+    return render(<NvisionNavbar/>, {wrapper: GlobalStateProvider});
```
Comment (emmaknor) [inline _tests_/front-end/components/nvisionNavbar.test.js:12]: I appreciate the wrapper. nice and DRY

File: _tests_/front-end/components/waterHistory.test.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 import {render} from '@testing-library/react';
 import '@testing-library/jest-dom';
 import * as React from 'react';
-import WaterHistory from 'waterHistory.jsx';
+import WaterDaily from 'waterDaily.jsx';
```
Comment (emmaknor) [inline _tests_/front-end/components/waterHistory.test.js:4]: nice format

**Review by emmaknor — APPROVED** [review]

**Pull request merged**